boards/arm/beaglebone_ai64/bbai_64.dts

```diff
@@ -0,0 +1,25 @@
+/* Copyright (C) 2023 BeagleBoard.org Foundation
+ * Copyright (C) 2023 S Prashanth
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <ti/j721e_r5.dtsi>
+#include <freq.h>
+
+/ {
+	chosen {
+		zephyr,sram = &atcm;
+		zephyr,console = &uart0;
+	};
+};
+
+&uart0 {
+	current-speed = <115200>;
+	status = "okay";
+	/* pinctrl-0 = <&uart0_default>;
+	 * pinctrl-names = "default";
+	 */
```

Comment:
Any reason for skipping this, is it due to missing RAT configurations?

Reply:
Added pinctrl modules